reconcile/gitlab_housekeeping.py

```
@@ -425,7 +425,7 @@ def handle_stale_items(
     items: Iterable[ProjectIssue | ProjectMergeRequest],
     item_type: str,
 ) -> None:
-    LABEL = "stale"  # noqa: N806
+    LABEL = "stale"  # ruff: ignore[non-lowercase-variable-in-function]
```

Ruff (0.16.0)

[warning] Invalid rule code in suppression: non-lowercase-variable-in-function

Enable `lint.preview` to use rule names

(RUF102)